src/app/system/account-number-preferences/view-account-number-preference/view-account-number-preference.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+  <button mat-raised-button color="primary" [routerLink]="['edit']" >
+      <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+      Edit
+  </button>
+
+  <button mat-raised-button color="warn" (click)="delete()">
+      <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+      Delete
+  </button>
+
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="accountNumberPreferenceForm">
```

Comment:
Don't use forms in view layout. Should be similar to view layouts in `accounting` section.